migrator/migrations/m_189_to_m_190_plop_add_closed_time_column/migration_impl.go

```diff
@@ -0,0 +1,70 @@
+package m189tom190
+
+import (
+	"github.com/pkg/errors"
+	updatedSchema "github.com/stackrox/rox/migrator/migrations/m_189_to_m_190_plop_add_closed_time_column/schema"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+	"github.com/stackrox/rox/pkg/utils"
+	"gorm.io/gorm/clause"
+)
+
+var (
+	batchSize = 2000
+	log       = logging.LoggerForModule()
+)
+
+func migrate(database *types.Databases) error {
```

Comment:
Do you have a ballpark of how long this will take? I would test with at least something like half a million and see how long this takes

Reply:
Ran 2M in a unit test.  Took 2:47

Reply:
Ok, that's not so bad 👍 thanks for the test